agent/proto/agent.proto

```diff
@@ -42,7 +43,13 @@ message WorkspaceApp {
 	}
 	Health health = 12;
 	bool hidden = 13;
-	string open_in = 14;
+
+	enum OpenIn {
+		WINDOW = 0;
+		SLIM_WINDOW = 1;
+		TAB = 2;
+	}
+	OpenIn open_in = 14;
```

Comment:
`open_in` affects the way application _clients_ (e.g. browsers) behave, so I don't think the Agent cares about this. Therefore I don't think we need to add this to the proto, as it would trigger a version bump.

Reply:
Sorry all, I missed that the other PR already changed `agent.proto`.  We haven't shipped anything yet, so no harm done, but we need to either:

1. bump the agent/tailnet API version in light of the new feature/field, or
2. revert that addition to `agent.proto`

Given that the agent doesn't need this information for anything, I prefer the latter.

Reply:
I was curious why this was already in the agent, but thought it was an old artefact. Since it was added recently, I second Spike's suggestion.